### PR TITLE
New nosql metric

### PR DESCRIPTION
**Pull request by voellm**
This pull requests adds new metrics for the Redis and Aerospike benchmarks. What those benchmarks publish today is a latency vs throughput series. That is very useful. What this adds is a decision point of publishing the max throughput using 1ms.

1ms was chosen because this is how many users rate and tune the system.

This does not modify the benchmark or any of the existing published metrics.

### CODE REVIEW

**Comment by voellm** [comment]
PTAL

File: perfkitbenchmarker/benchmarks/aerospike_benchmark.py [hunk 1]
```diff
@@ -188,6 +189,13 @@ def ParseOutput(output):
     if FLAGS.aerospike_storage_type == DISK:
       metadata['Disk Type'] = 'Local' if FLAGS.use_local_disk else 'Remote'
     samples.append(sample.Sample('Average Latency', latency, 'ms', metadata))
+    if latency < 1.0:
+      max_throughput_for_completion_latency_under_1ms = tps
```
Comment (cmccoy) [inline perfkitbenchmarker/benchmarks/aerospike_benchmark.py]: Is `tps` guaranteed to be monotonically increasing (that is: could `max_throughput_for_completion_latency_under_1m` ever decrease here)?

Comment (voellm) [inline perfkitbenchmarker/benchmarks/aerospike_benchmark.py]: That is a very good point.  Indeed it may not be.  Let me fix this.

**Comment by voellm** [comment]
PTAL - Did a full run of Aerospike and Redis.

**Comment by cmccoy** [comment]
LGTM.

**Pull request merged**